projects/plugins/jetpack/modules/masterbar/admin-menu/load.php

```diff
@@ -2,22 +2,30 @@
 /**
  * Admin Menu loader.
  *
+ * @deprecated $$next-version$$
+ *
  * @package Jetpack
  */
 
 namespace Automattic\Jetpack\Dashboard_Customizations;
 
+_deprecated_file( __FILE__, 'jetpack-$$next-version$$' );
+
+use Automattic\Jetpack\Masterbar\Base_Admin_Menu;
 use Automattic\Jetpack\Status\Host;
 use Automattic\Jetpack\Tracking;
 
 /**
  * Checks whether the navigation customizations should be performed for the given class.
  *
+ * @deprecated $$next-version$$
+ *
  * @param string $admin_menu_class Class name.
  *
  * @return bool
  */
 function should_customize_nav( $admin_menu_class ) {
+	_deprecated_function( __FUNCTION__, 'jetpack-$$next-version$$', 'Automattic\\Jetpack\\Masterbar\\should_customize_nav' );
 	// Make sure the class extends the base admin menu class.
 	if ( ! is_subclass_of( $admin_menu_class, Base_Admin_Menu::class ) ) {
```

Comment:
I struggled a lot on the best approach for the current file but also the `bootsrtap.php` ones. I don't think we should return for example `Masterbar\should_customize_nav` here because there's no guarantee the corresponding `load.file` in the `jetpack-masterbar` package has been loaded 🤷 

I played around with requiring that file here but this breaks the unit tests because of `jetpack-vendor` not being present under the context of unit tests. I think it's an overkill to make this work but I'm open to suggestions.

Reply:
It makes sense to me to leave the code in the relevant functions for now given those issues.

Reply:
`jetpack_vendor` should be there during the phpunit tests, if we didn't run `composer install` then phpunit itself wouldn't be there with the way we run it in CI. Do you have a link to a run where it failed when you tried requiring the package file?

Reply:
Yeah, this all originated from the same `jetpack-vendor` typo you already pointed out. But still, I think requiring the package file instead is not a good approach in this case since the namespaces are different.